hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/om/TransactionInfoRepair.java

```diff
@@ -51,8 +51,10 @@
 )
 public class TransactionInfoRepair extends RepairTool {
 
-  @CommandLine.ParentCommand
-  private RDBRepair parent;
+  @CommandLine.Option(names = {"--db"},
+          required = true,
+          description = "Database File Path")
```

Comment:
nit: let's keep continuation indent as 4 spaces

```suggestion
      required = true,
      description = "Database File Path")
```